### PR TITLE
Replace --bootstrap-script with --bootstrap-command-file

### PR DESCRIPTION
**Pull request by fmoor**
This is a follow up to https://github.com/edgedb/edgedb/pull/4694

### CODE REVIEW

File: docker-entrypoint-funcs.sh [hunk 1]
```diff
@@ -363,11 +363,25 @@ edbdocker_setup_env() {
...
   : "${EDGEDB_SERVER_COMPILER_POOL_SIZE:=}"
 
+  if [ -n "${EDGEDB_SERVER_BOOTSTRAP_SCRIPT_FILE:-}" ]; then
+    if [ -n "${EDGEDB_SERVER_BOOTSTRAP_COMMAND_FILE}" ]; then
+      edbdocker_die "ERROR: EDGEDB_SERVER_BOOTSTRAP_COMMAND_FILE and EDGEDB_SERVER_BOOTSTRAP_SCRIPT_FILE are mutually exclusive, but both are set"
+    else
+      msg=(
+        "============================================================"
+        "WARNING: EDGEDB_SERVER_BOOTSTRAP_SCRIPT_FILE is deprecated. "
+        "         Use EDGEDB_SERVER_BACKEND_DSN instead."
```
Comment (elprans) [inline docker-entrypoint-funcs.sh:377]:
```suggestion
        "         Use EDGEDB_SERVER_BOOTSTRAP_COMMAND_FILE instead."
```

File: docker-entrypoint-funcs.sh [hunk 2]
```diff
@@ -752,18 +762,11 @@ edbdocker_bootstrap_instance() {
...
-      bootstrap_opts+=(--bootstrap-script="${EDGEDB_SERVER_BOOTSTRAP_SCRIPT_FILE}")
-    fi
-
-  elif [ -n "${EDGEDB_SERVER_BOOTSTRAP_COMMAND}" ]; then
+  if [ -n "${EDGEDB_SERVER_BOOTSTRAP_COMMAND}" ]; then
     bootstrap_opts+=(--bootstrap-command="${EDGEDB_SERVER_BOOTSTRAP_COMMAND}")
 
   elif [ -e "/edgedb-bootstrap.edgeql" ]; then
-    bootstrap_opts+=(--bootstrap-script="/edgedb-bootstrap.edgeql")
+    bootstrap_opts+=(--bootstrap-command-file="/edgedb-bootstrap.edgeql")
```
Comment (elprans) [inline docker-entrypoint-funcs.sh:769]: You need to check if the server actually supports the new spelling with `edbdocker_server_supports`

File: docker-entrypoint-funcs.sh [hunk 3]
```diff
@@ -115,12 +115,12 @@ edbdocker_parse_args() {
         EDGEDB_SERVER_BOOTSTRAP_COMMAND="${1#*=}"
         shift
         ;;
-      --bootstrap-script)
-        _edbdocker_parse_arg "EDGEDB_SERVER_BOOTSTRAP_SCRIPT_FILE" "$1" "$2"
+      --bootstrap-command-file)
```
Comment (elprans) [inline docker-entrypoint-funcs.sh:118]: Old spellings must still be supported here.

Comment (fmoor) [reply docker-entrypoint-funcs.sh:126]: changed

**Review by elprans — APPROVED** [review]

**Pull request merged**